for_each_test.go

```diff
@@ -55,3 +56,39 @@ func TestForEachContext(t *testing.T) {
 		require.Equal(t, testCase.Out, stringSlice)
 	}
 }
+
+func TestForEachContextError(t *testing.T) {
+	type contextKey string
```

Comment:
We have that declaration already in the package level (in `for_each_test.go` I guess) and, thus, we don't need to declare it again ; )

```suggestion
```